src/core/json-schema-components.jsx

```diff
@@ -40,6 +40,8 @@ export class JsonSchemaForm extends Component {
     const { dispatchInitialValue, value, onChange } = this.props
     if(dispatchInitialValue) {
       onChange(value)
+    } else if(dispatchInitialValue === false) {
```

Comment:
is this required: `if(dispatchInitialValue === false)`. If not, we could just use else { ... }. Or more node-style, 
```
if (!dispatchInitialValue) {
...new 
return 
}
...ori
```

Reply:
yes this is required since dispatchInitialValue is oas3 specific and in case of <oas3(body param) it will not be provided.

Reply:
In addition to that there were several failing tests that indicated that when using else there is something wrong.